src/modules/endpoint/models/__tests__/CategoriesMapModel.spec.js

```diff
@@ -21,26 +23,49 @@ describe('CategoriesMapModel', () => {
   })
 
   const categories = [
-    new CategoryModel({id: 0, path: '/augsburg/de', title: 'augsburg', parentPath: ''}),
+    new CategoryModel({
+      id: 0,
+      path: '/augsburg/de',
+      title: 'augsburg',
+      parentPath: '',
+      availableLanguages: new Map(),
+      content: 'exampleContent0',
+      lastUpdate: moment(),
```

Comment:
Not sure if this is important but using `moment()` here makes the tests a bit non-deterministic. Maybe a fixed date and time would be better.